homeassistant/components/auth/login_flow.py

```diff
@@ -257,7 +257,7 @@ async def get(self, request):
 
     @RequestDataValidator(vol.Schema({"client_id": str}, extra=vol.ALLOW_EXTRA))
     @log_invalid_auth
-    async def post(self, request, flow_id, data):
+    async def post(self, request, data, flow_id):
```

Comment:
Could it have been typed here?

Reply:
Opened #75586 to fully type the `auth` integration.